File: 1446-AngleBetweenHandsOfAClock/1446-AngleBetweenHandsOfAClock.cpp
// Last updated: 6/18/2026, 4:21:48 PM
class Solution {
public:
    double angleClock(int hour, int minutes) {
        double result = abs((30*hour) - (5.5*minutes));
        return (result > 180) ? 360-result: result;
    }
};

File: 1446-AngleBetweenHandsOfAClock/1446-AngleBetweenHandsOfAClock.java
// Last updated: 6/18/2026, 4:21:45 PM
class Solution {
    public double angleClock(int h, int m) {
        double d = Math.abs(30*h - 5.5*m);
        return (d > 180)?360-d:d;
    }
}